app/views/spina/journal/primer_theme/issues/_article.html.haml

```diff
@@ -0,0 +1,15 @@
+= render(Primer::FlexComponent.new(direction: [:column, nil, :row, nil])) do
+  = render(Primer::FlexItemComponent.new(flex_auto: true)) do
+    = render(Primer::HeadingComponent.new(tag: :h3, mb: 1)) do
+      = link_to article.title, frontend_issue_article_path(article.issue, article)
+    = render(Primer::TextComponent.new(tag: :div, color: :text_secondary)) do
+      = render(Primer::OcticonComponent.new(article.affiliations.many? ? 'people' : 'person'))
+      = render(Primer::BaseComponent.new(tag: :address, display: :inline)) do
+        = article.affiliations.collect(&:name).to_sentence
+    - unless article.doi.blank?
+      = render(Primer::TextComponent.new(tag: :div, color: :text_secondary)) do
+        = link_to nil, article.doi
```

Comment:
Is this right?

Reply:
This is intended - initial `nil` argument of `link_to` prints the link itself as the text of the anchor tag.